rai/node/rpc.cpp

```diff
@@ -2869,6 +2869,64 @@ void rai::rpc_handler::search_pending_all ()
 	}
 }
 
+void rai::rpc_handler::request_reconfirmation ()
+{
+	std::string hash_text (request.get <std::string> ("hash"));
+	rai::block_hash hash_l;
+	if (!hash_l.decode_hex (hash_text))
+	{
+		rai::transaction transaction (node.store.environment, nullptr, false);
+		auto block_l (node.store.block_get (transaction, hash_l));
+		if (block_l != nullptr)
+		{
+			node.request_reconfirmation(std::shared_ptr<rai::block>(move(block_l)));
+		}
+		else
+		{
+			error_response (response, "Block not found");
+		}
+	}
+	else
+	{
+		error_response (response, "Invalid block hash");
+	}
+}
+
+void rai::rpc_handler::check_election_results ()
+{
+	std::string hash_text (request.get <std::string> ("hash"));
+	rai::block_hash hash_l;
+	if (!hash_l.decode_hex (hash_text))
+	{
+		rai::transaction transaction (node.store.environment, nullptr, false);
+		auto block_l (node.store.block_get (transaction, hash_l));
+		if (block_l != nullptr)
+		{
+			rai::election_result result;
+			if (!node.check_election_results (std::shared_ptr <rai::block>(move (block_l)), result))
+			{
+				boost::property_tree::ptree response_l;
+				response_l.put ("verified", result.met_reconfirm_threshold && result.confirmed);
```

Comment:
I worry that the presence of this field will negate the flexibility benefit of the "percent" below.  We should assume that clients are the lowest common denominator and won't take the time to understand what "verified" means for their particular transaction.

Reply:
I agree, that's how I had it in a previous version of the patch, then was worried about not providing some sort of official view on the level of risk. I'll revert.